### PR TITLE
NOMERGE: Home page styled + identity created/teaked

### PR DESCRIPTION
**Pull request by candroo**
Still need to make this mobile friendly (very straight forward).
The logo has been tweaked but not finalised. It has stronger type, less rounded corners and bolder colours.

(Just noticed it's missing the radify logo SVG, will add this in later)

screen shot 2015-05-19 at 11 58 25

### CODE REVIEW

**Comment by gavD** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-    <a href="https://github.com/radify/PR.js">GitHub</a> |
-    <a href="http://radify.io">
-      This product proudly hand-crafted by
-      <img src="https://pbs.twimg.com/profile_images/595576094838890496/aJGJ3RBo_normal.png" alt="Radify" />
-    </a>
-  </footer>
+	<div ui-view></div>
+	
+	<footer>
+		<!--<a href="https://github.com/radify/PR.js">GitHub</a> | -->
```
Comment (nateabele) [inline index.html]: We still need the GitHub link in here somewhere.

**Comment by nateabele** [comment]
Just tested this locally. The only issue seems to be that the UI gets cut off if the page is stretched wide. I'm going to blindly assume this will be fixed in your mobile layout changes. :smile:

**Comment by candroo** [comment]
Home page should be all done.

With regards to the rest of the app, I've messed up the UI somewhat in that it's now a scrolling page. Ill try and look at his tomorrow.

Other than that, all the styling is done. Should look something like...

screen shot 2015-05-20 at 01 39 35

**Comment by candroo** [comment]
The layout has now been updated too.
It now has a fixed results pane, 100% height, scrollable lists etc.

screen shot 2015-05-20 at 09 46 03

**Comment by nateabele** [comment]
@candroo Tested both pages. All looks well except for the issue where the search box disappears if the browser is too short/wide.

**Comment by candroo** [comment]
Oh, weird. What browser is that?

**Comment by candroo** [comment]
Ok, I see the problem.

**Comment by candroo** [comment]
@nateabele that should be fixed now (I hope)

**Pull request merged**